src/deploy/table_location.sql

```diff
@@ -0,0 +1,50 @@
+BEGIN;
+
+CREATE TABLE maevsi.location (
+  id         UUID PRIMARY KEY,
+  location_type CHAR(1) NOT NULL,
+  latitude   DOUBLE PRECISION NOT NULL,
+  longitude  DOUBLE PRECISION NOT NULL,
+  -- possible future extension (using PostGIS):
+  -- geom    GEOMETRY
+
+  CHECK (location_type IN ('A', 'E'))
+);
+
+COMMENT ON TABLE maevsi.location IS 'Location data based on GPS coordnates.';
+COMMENT ON COLUMN maevsi.location.id IS 'The locations''s internal id.';
+COMMENT ON COLUMN maevsi.location.location_type IS 'The type of the location (A = account, E = event)';
+COMMENT ON COLUMN maevsi.location.latitude IS 'reference to an account (if not null).';
+COMMENT ON COLUMN maevsi.location.longitude IS 'reference to an account (if not null).';
+
+GRANT SELECT, INSERT, UPDATE, DELETE ON TABLE maevsi.location TO maevsi_account;
+GRANT SELECT ON TABLE maevsi.location TO maevsi_anonymous;
+
+/*
+-- If PostGIS is used:
```

Comment:
I think that we'll want to query events in a certain radius, so PostGIS should be used. It can't be enabled in the debezium Docker image that we're using right away though, it seems. I'll take a look.

Reply:
Ah, nvm, [PostGIS seems to be packaged](https://github.com/debezium/container-images/blob/88532c5729458827cc4dc6d1b390ab580b1c31b9/postgres/17/Dockerfile#L3C5-L3C16) already? Didn't see it in the list of available extensions.

Reply:
Ahh, we're using [the alpine variant](https://github.com/debezium/container-images/blob/c440e4cffd2a9d3c2c02465fb3182ce32a96dfe2/postgres/17-alpine/Dockerfile) and this does not include PostGIS. The alpine variant did in fact include PostGIS up until 2019 when it was removed with [DBZ-1549](https://issues.redhat.com/browse/DBZ-1549) in https://github.com/debezium/container-images/commit/dd490a0af12e65d00329de9e4a0f247963655205.

@sthelemann I switched to the non-alpine variant so PostGIS is available on the `beta` branch in our stack now (https://github.com/maevsi/maevsi_stack/pull/157).

Reply:
@sthelemann you can rebase this PR on the beta branch or merge the beta branch into your feature branch, I think